web/modules/custom/dpl_related_content/dpl_related_content.module

```diff
@@ -116,6 +117,15 @@ function _dpl_related_content_filter_paragraph(array $variables, string $list_st
       $service->andConditions = ($paragraph->get('field_filter_cond_type')->getString() == 'and');
     }
 
+    if ($paragraph->hasField('field_filter_content_types')) {
+      // Turning 'xx, yy, zz' into an array.
+      $content_types = explode(', ', $paragraph->get('field_filter_content_types')->getString());
```

Comment:
Consider making the string parsing more flexible.

Splitting by very specific formatting `xx, yy, zz` can be fragile. Consider doing something like this:

```php
$content_types = preg_split('/\s*,\s*/', $paragraph->get('field_filter_content_types')->getString(), flags: PREG_SPLIT_NO_EMPTY)`
```